libraries/Microsoft.Bot.Builder/Teams/TeamsTurnContextExtensions.cs

```diff
@@ -44,6 +44,40 @@ public static async Task<(ConversationReference conversationReference, string ac
             return (conversationReference, conversationResourceResponse.ActivityId);
         }
 
+        public static async Task TeamsCreateOneToOneConversationAsync(this ITurnContext turnContext, ChannelAccount account, Activity message, CancellationToken cancellationToken = default)
```

Comment:
How about we just overload the method name "TeamsCreateConversationAsync"?
(naming it so directly after the scenario just seems to add more new names.)

And presumably this method should return the tuple of ConverstaionReference and ActivityId.